src/common.ts

```diff
@@ -29,7 +29,7 @@ export const permissions = [
     /* 4 */ _("Read-only"),
     /* 5 */ _("Read and execute"),
     /* 6 */ _("Read and write"),
-    /* 7 */ _("Read, write and execute"),
+    /* 7 */ _("Read, write, and execute"),
```

Comment:
Sneaky!

Reply:
"Oxford comma" (serial comma) is usually [unambiguous](https://en.wikipedia.org/wiki/Serial_comma#Ambiguity). :grin:

Since this is UI text, we could also just do lists without the "and"s too:

- Read-only
- Read, execute
- Read, write
- Read, write, execute

(I did consider this also, but went the Oxford route.)